src/main/java/tasklist/TaskList.java

```diff
@@ -0,0 +1,324 @@
+package tasklist;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+import java.util.List;
+import java.util.Scanner;
+import tasks.Deadline;
+import tasks.Event;
+import tasks.Task;
+import tasks.ToDo;
+import exceptions.TheOrangeRatchetCatException;
+
+public class TaskList {
+
+    /**
+     * Prints out all tasks currently in the list
+     *
+     * @param items represents the list of task
+     * @param scanner the scanner object to read next line of input
+     * @return returns the next input command by user
+     */
+    public static String checkList(List<Task> items, Scanner scanner) {
```

Comment:
Is there a reason why scanner is part of the arguments but not used in the function body? And is there a better name to be given for checkList based on your javadocs descriptor?